### PR TITLE
Facet router

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

**Comment by mareklibra** [comment]
Can you please split package.json/yarn.lock changes to a separate commit within this PR?

File: src/components/Router.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import { Clusters, ClusterPage } from './clusters';
+import { store } from '../store';
+import { routeBasePath } from '../config';
```
Comment (jtomasek) [inline src/components/Router.tsx:6]: Nit
```suggestion
import { routeBasePath } from '../config/constants';
```

Comment (rawagner) [reply src/components/Router.tsx]: fixed

**Comment by rawagner** [comment]
@mareklibra rebased, no changes in yarn.lock

**Pull request merged**